server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -291,7 +298,7 @@ module.exports = {
       {
         title: 'BASS address',
         label: bassAddress.getLabel({ decisions, tasks }),
-        action: bassAddress.getCaAction({ tasks }),
+        action: bassAddress.getCaAction({ tasks }, 'bass-address'),
```

Comment:
can this second parameter be got rid of too?

Reply:
yep - just missed it
done